src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -189,7 +189,7 @@ public bool CrossCompilerAvailable (string supportedAbis)
 				string v = Path.GetFileName (dir).Replace ("v", "");
 				if (!Version.TryParse (v, out version))
 					continue;
-				if (latest.Major < version.Major && latest.Minor <= version.Minor) {
+				if (latest.Major < version.Major || (latest.Major == version.Major && latest.Minor <= version.Minor)) {
```

Comment:
Does System.Version have a > operator we can just use here?

Reply:
I'll give that a test.